README.md

```diff
@@ -368,7 +368,18 @@ You can also manually specify your multi tenancy configuration with a hash
 ```ruby
 client.schema.create(
     # Other keys...
-    mutli_tenant: { enabled: true, autoTenantCreation: true, autoTenantActivation: true },
+    multi_tenant: { enabled: true, autoTenantCreation: true, autoTenantActivation: true },
+)
+```
+
+Added shortcuts for `multiTenancyConfig`:
+
+```ruby
+client.schema.create(
+    # Other keys...
+    multi_tenant: true,
+    auto_tenant_creation: true,
+    auto_tenant_activation: true
```

Comment:
What will actually get passed in to Weaviate if you do
```ruby
client.schema.create(
    multi_tenant: { enabled: true, autoTenantCreation: true, autoTenantActivation: true },
    auto_tenant_creation: false,
    auto_tenant_activation: false
```

Can autoTenantCreation == true or autoTenantActivation == true but enabled: false?


  multi_tenant: { enabled: true, autoTenantCreation: true, autoTenantActivation: true }